qa/TL1_jupyter_GDS/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+# used pip packages
+pip_packages="jupyter matplotlib numpy"
+target_dir=./docs/examples/
+
+test_body() {
+    test_files=("general/data_loading/numpy_reader.ipynb")
+
+    # GDS can't read data from the Docker filesystem.
+    # Here we are copying the relevant part of DALI_extra that we need to run the notebook
+    # to a "real" filesystem
```

Comment:
You can write that `scratch` is assume to be a real FS. If it is run on the real FW there is no need for this copy.